content/en/tests/containers.md

```diff
@@ -19,283 +19,387 @@ If you run your tests inside a container that you launch yourself within the bui
+| `TRAVIS_APP_HOST`            | The name of the server compiling the build script.                                                   |
+| `TRAVIS_COMMIT_RANGE`        | The range of commits included in the push or pull request.                                           |
+| `TRAVIS_COMPILER`            | Indicates the compiler used by the current job (such as clang, gcc).                                   |
+| `TRAVIS_DEBUG_MODE`          | Set to `true` if the job is running in debug mode.                                                    |
+| `TRAVIS_DIST`                | Indicates the distribution the current job is running on.                                            |
+| `TRAVIS_EVENT_TYPE`          | Indicates how the build was triggered. Possible values are `push`, `pull_request`, `api`, `cron`.   |
+| `TRAVIS_JOB_ID`              | The ID of the current job used internally by Travis CI.                                              |
+| `TRAVIS_JOB_NAME`            | The job name if it was specified, or `""`.                                                             |
+| `TRAVIS_JOB_NUMBER`          | The number of the current job (such as “4.1”).                                                          |
+| `TRAVIS_OS_NAME`             | On multi-OS builds, this value indicates the platform the job is running on (such as `linux`, `osx`). |
+| `TRAVIS_CPU_ARCH`            | On multi-arch builds, this value indicates the CPU architecture (such as `amd64`, `arm64`).           |
+| `TRAVIS_OSX_IMAGE`           | The `osx_image` value configured in `.travis.yml`.                                                    |
+| `TRAVIS_PULL_REQUEST`        | The pull request number if the current job is a pull request, `false` if it’s not a pull request.    |
+| `TRAVIS_PULL_REQUEST_SHA`    | The commit SHA of the HEAD commit of the PR if the job is a pull request.                            |
+| `TRAVIS_PULL_REQUEST_IS_DRAFT` | Set to `true` if the PR is in a draft state, `false` if it is not.                                    |
+| `TRAVIS_SECURE_ENV_VARS`     | Set to `true` if there are any encrypted environment variables, `false` otherwise.                   |
+| `TRAVIS_SUDO`                | `true` or `false` based on whether `sudo` is enabled.                                                 |
+| `TRAVIS_TEST_RESULT`         | `0` if all commands in the script section have exited with zero, `1` otherwise.                     |
+| `TRAVIS_BUILD_STAGE_NAME`    | The build stage. If a build does not use build stages, this variable is empty.                       |
+| `TRAVIS_JOB_RESTARTED`       | Set to `true` if the build is restarted, `false` otherwise.                                           |
+| `TRAVIS_JOB_RESTARTED_BY`    | The login of the user triggering the restart.                                                         |
```

Comment:
```suggestion
| Variable                     | Description                                                                                           |
|------------------------------|-------------------------------------------------------------------------------------------------------|
| `TRAVIS`                     | Always set to `true` to indicate that the build is running on Travis CI.                              |
| `TRAVIS_BUILD_ID`            | The ID of the current build used internally by Travis CI.                                             |
| `TRAVIS_BUILD_NUMBER`        | The number of the current build. For example: `4`.                                                    |
| `TRAVIS_BUILD_WEB_URL`       | URL to the build log.                                                                                 |
| `TRAVIS_BUILD_DIR`           | The absolute path to the directory where the repository being built has been copied on the worker.    |
| `TRAVIS_JOB_WEB_URL`         | URL to the job log.                                                                                   |
| `TRAVIS_REPO_SLUG`           | The slug (in form: `owner_name/repo_name`) of the repository currently being built.                   |
| `TRAVIS_COMMIT`              | The commit that the current build is testing.                                                         |
| `TRAVIS_BRANCH`              | For push builds, the name of the branch. For PR builds, the name of the branch targeted by the PR.    |
| `TRAVIS_TAG`                 | If the current build is for a git tag, this variable is set to the tag’s name, otherwise it is empty. |
| `TRAVIS_PULL_REQUEST_SLUG`   | If the current job is a pull request, the slug of the repository from which the PR originated.        |
| `TRAVIS_PULL_REQUEST_BRANCH` | If the current job is a pull request, the name of the branch from which the PR originated.            |
| `TRAVIS_COMMIT_MESSAGE`      | The commit subject and body, unwrapped.                                                               |

```